packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -200,6 +332,68 @@ _CSS Classes and CSS Custom Properties that affect the component_
 
 _Note_: This initial component design serves as a starting point for implementation, and it may undergo changes once the visual design is completed.
 
+#### User mention element:
+
+This element serves as a container for the mapping elements and additionally functions as a configuration element. It allows the transmission of
+arbitrary values that are specific to the `@mention` users within the rich text components.
+
+_Component Name_
+
+-   `nimble-rich-text-mention-users`
+
+_Content_
+
+-   One or more `nimble-mapping-mention-user` elements
+
+#### Mapping element (mention):
+
+This mapping element is employed to establish a connection between the value displayed in the mapping view and the corresponding value stored within
+the markdown string. For instance, the username for an `@mention` is contained in the `text` attribute, which is used for display in the mention
+view, while the user ID is contained in the `key` attribute which is used to store within the markdown string.
+
+_Component Name_
+
+-   `nimble-mapping-mention-user`
+
+_Props/Attrs_
+
+-   `key`: string | number | boolean
```

Comment:
Do we really need to support `boolean` type? If yes, provide a few examples or brief on why we need it.

Reply:
This type is supported by the base mapping class. That's why we added a validation method to check the types of `key` attributes. If it is invalid, the option will be empty in the dropdown list.